docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -225,6 +225,24 @@ Note: Actually, Gitea supports only SMTP with STARTTLS.
 - `MAX_GIT_DIFF_FILES`: Max number of files shown in diff view.
 - `GC_ARGS`: Arguments for command `git gc`, e.g. `--aggressive --auto`.
 
+## markup (`markup`)
```

Comment:
Tiny nit, all other titles in this document start with a capital letter. Could you change this so that the first word in this title starts with `M`?

Other than that, LGTM.

Reply:
Sorry, just see this after merged. I will send another PR to fix this.